var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -539,6 +547,9 @@ def install(self, spec, prefix):
         if self.spec.satisfies("@5.3.0:"):
             openmp_common_args += ["-DLIBOMPTARGET_ENABLE_DEBUG=ON"]
 
+        if self.spec.satisfies("@6.1:"):
```

Comment:
I don't believe we need this block. Cuda plugin is still built with ROCm 6.1 just not the supporting bitcode files for hostexec and devicertl. I was suggesting we just build what is in 6.1. This would disable the build of the cuda plugin entirely.

Reply:
Hi @estewart08 , Thanks for the review. Removed the block setting LIBOMPTARGET_BUILD_CUDA_PLUGIN=OFF.